etna/whatson/models.py

```diff
@@ -0,0 +1,621 @@
+    """
+    This model is used to add sessions to an event
+    e.g. 28th September @ 9:00, 29th September @ 10:30, 30th September @ 12:00.
+    These will link to the Eventbrite page.
+    """
+
+    page = ParentalKey(
+        "wagtailcore.Page",
+        on_delete=models.CASCADE,
+        related_name="sessions",
+    )
+
+    start = models.DateTimeField(
+        verbose_name=_("starts at"),
+        blank=False,
+    )
+
+    end = models.DateTimeField(
+        verbose_name=_("ends at"),
+        blank=False,
+    )
+
+    panels = [
+        FieldPanel("start"),
+        FieldPanel("end"),
+    ]
+
+    class Meta:
+        verbose_name = _("session")
+        verbose_name_plural = _("sessions")
+        ordering = ["start"]
+
+
+class WhatsOnPage(BasePageWithIntro):
+    """WhatsOnPage
+
+    A page for listing events.
+    """
+
+    @cached_property
+    def events(self):
+        """
+        Returns a queryset of events that are children of this page.
+        """
+        return (
+            EventPage.objects.child_of(self)
+            .live()
+            .public()
+            .order_by("start_date")
+        )
+
+    # DataLayerMixin overrides
+    gtm_content_group = "What's On"
+
+    class Meta:
+        verbose_name = _("What's On page")
+
+    parent_page_types = [
+        "home.HomePage",
+    ]
+    subpage_types = [
+        "whatson.EventPage",
+    ]
+
+    max_count = 1
+
+    content_panels = BasePageWithIntro.content_panels
+
+
+class EventPage(ArticleTagMixin, TopicalPageMixin, BasePageWithIntro):
+    """EventPage
+
+    A page for an event.
+    """
+
+    # Content
+    lead_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    # Event information
+    event_type = models.ForeignKey(
+        EventType,
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    start_date = models.DateTimeField(
+        verbose_name=_("start date"),
+        null=True,
+        blank=True,
+        editable=False,
+    )
+
+    end_date = models.DateTimeField(
+        verbose_name=_("end date"),
+        null=True,
+        blank=True,
+        editable=False,
+    )
+
+    description = RichTextField(
+        verbose_name=_("description"),
+        blank=True,
+        help_text=_("A description of the event."),
+    )
+
+    useful_info = RichTextField(
+        verbose_name=_("need to know"),
+        blank=True,
+        help_text=_("Useful information about the event."),
+    )
+
+    target_audience = RichTextField(
+        verbose_name=_("who it's for"),
+        blank=True,
+        help_text=_("Info about the target audience for the event."),
+    )
+
+    # Venue information
+    venue_type = models.CharField(
+        max_length=15,
+        verbose_name=_("venue type"),
+        null=False,
```

Comment:
`False` is always the default value for `null`, so you don't have to specify this here.